doc/source/serve/production.md

```diff
@@ -583,3 +565,217 @@ This section summarizes the best practices when deploying to production:
+
+Once the `RayService` is running, we can query it over HTTP using the service created by the controller.
+This service can be queried directly from inside the cluster, but to access it from your laptop you'll need to configure a Kubernetes ingress or use port forwarding as below:
+
+```console
+$ kubectl port-forward service/rayservice-sample-serve-svc 8000
+$ curl -X POST -H 'Content-Type: application/json' localhost:8000 -d '["MANGO", 2]'
+6
+```
+
+### Getting the status of the application
+
+As the `RayService` is running, the `KubeRay` controller continually monitors it and writes relevant status updates to the CR.
+You can view the status of the application using `kubectl describe`.
+This includes the status of the cluster, any events that happened such as health check failures or restarts, and the same application-level statuses reported by [`serve status`](serve-in-production-inspecting).
+
+```console
+$ kubectl get rayservices
+NAME                AGE
+rayservice-sample   7s
+
+$ kubectl describe rayservice rayservice-sample
+```
+
+### Updating the application
+
+To update the `RayService`, modify the manifest and apply it use `kubectl apply`.
+There are two types of updates that can occur:
+- *Application-level updates*: when only the Serve config options are changed, the update is applied _in-place_ on the same Ray cluster. This enables [lightweight updates](serve-in-production-lightweight-update) such as scaling a deployment up or down or modifying autoscaling parameters.
+- *Cluster-level updates*: when the `RayCluster` config options are changed, such as updating the container image for the cluster, it may result in a cluster-level update. In this case, a new cluster is started, the application is deployed to it, and once it is running the Kubernetes service is updated to point at the new cluster. There should not be any downtime for the application, but note that this requires the Kubernetes cluster to be large enough to schedule both Ray clusters.
+
+#### Example: Serve config update
+
+In the `FruitStand` example above, let's change the price of a mango in the Serve config to 4:
+
+```console
+  - name: MangoStand
+    numReplicas: 1
+    userConfig: |
+      price: 4
+```
+
+Now to update the application we apply the modified manifest:
+
+```console
+$ kubectl apply -f ray_v1alpha1_rayservice.yaml
+
+$ kubectl describe
+...
+  serveDeploymentStatuses:
+  - healthLastUpdateTime: "2022-07-18T21:51:37Z"
+    lastUpdateTime: "2022-07-18T21:51:41Z"
+    name: MangoStand
+    status: UPDATING
+...
+```
+
+If we query the application, we can see that we now get a different result reflecting the updated price:
+
+```console
+$ curl -X POST -H 'Content-Type: application/json' localhost:8000 -d '["MANGO", 2]'
+8
+```
+
+#### Updating the RayCluster config
+
+The process of updating the RayCluster config is the same as updating the Serve config.
+For example, we can update the number of worker nodes to 2 in the manifest:
```

Comment:
> For example, we can update the number of worker nodes to 2 in the manifest:

Can we link `worker nodes` to documentation explaining what a Ray worker node is? I don't think we explain that term anywhere in the Serve docs.

Reply:
added a link to the clusters key concepts doc.